packages/screens/AdministrationDashboard/CurrentlyHighLightedProject.tsx

```diff
@@ -0,0 +1,291 @@
+  FlatList,
+  Image,
+  ImageBackground,
+  StyleSheet,
+  TouchableOpacity,
+  View,
+  useWindowDimensions,
+} from "react-native";
+import {
+  Bars2Icon,
+  CheckBadgeIcon,
+  CheckCircleIcon,
+  ChevronDownIcon,
+  ChevronUpIcon,
+  PlusIcon,
+  TrashIcon,
+} from "react-native-heroicons/outline";
+
+import avaPNG from "../../../assets/default-images/ava.png";
+import guardianPng from "../../../assets/default-images/guardian_1.png";
+import { BrandText } from "../../components/BrandText";
+import { Box } from "../../components/boxes/Box";
+import { SearchInput } from "../../components/sorts/SearchInput";
+import {
+  neutralA3,
+  neutral17,
+  primaryColor,
+  neutral33,
+  neutral00,
+  gradientColorLightBlue,
+} from "../../utils/style/colors";
+import {
+  fontSemibold13,
+  fontSemibold14,
+  fontSemibold16,
+} from "../../utils/style/fonts";
+
+const MD_BREAKPOINT = 820;
+
+type Props = {
+  setIsEditHighlighted: (val: boolean) => void;
+};
+
+export const CurrentlyHighLightedProject = ({
+  setIsEditHighlighted,
+}: Props) => {
+  const { width } = useWindowDimensions();
+
+  const [guardiansHandler, setGuardiansHandler] = useState<any>([]);
+  const [isListOpen, setIsListOpen] = useState<any>();
+
+  return (
+    <View style={{ marginVertical: 32 }}>
+      <TouchableOpacity
+        onPress={() => setIsEditHighlighted(false)}
+        style={{ alignSelf: "flex-start" }}
+      >
+        <Box notched style={styles.boxBtn}>
+          <CheckCircleIcon width={20} height={20} />
+          <BrandText
+            style={[fontSemibold13, { color: neutral17, marginLeft: 5 }]}
+          >
+            Save changes
+          </BrandText>
+        </Box>
+      </TouchableOpacity>
+      <View style={styles.marginVertical24}>
+        <BrandText style={[fontSemibold14, { color: neutralA3 }]}>
+          Select the desired collections to display and drag them in the desired
+          order.
+        </BrandText>
+      </View>
+      <View style={[{ flex: 1 }, styles.marginVertical24]}>
+        <View
+          style={{ flexDirection: width >= MD_BREAKPOINT ? "row" : "column" }}
+        >
+          {guardiansHandler?.map((d: any, i: any) => {
+            return (
+              <>
+                <View
+                  style={[
+                    styles.viewBox,
+                    { width: width >= MD_BREAKPOINT ? "19%" : "100%" },
+                  ]}
+                >
+                  <View style={styles.insideBoxMap}>
+                    <View style={styles.dotBackground}>
+                      <BrandText style={[fontSemibold14]}>{i + 1}</BrandText>
+                    </View>
+                    <View>
+                      <Bars2Icon color="white" />
+                    </View>
+                  </View>
+                  <Box notched style={styles.herosLisBox}>
+                    <TouchableOpacity
+                      onPress={() =>
+                        setIsListOpen((o: any) => (o === i ? null : i))
```

Comment:
We should avoid `too short` var name, it makes the code harder to read and understand

Reply:
Fixed, PR updated